extensions/wikia/InstantGlobals/InstantGlobalsApiController.class.php

```diff
@@ -12,6 +12,21 @@ public function getNewsAndStoriesVariables() {
 		$this->response->setData( $instantGlobals );
 	}
 
+	/**
+	 * Get news&stories tracking opt-outs values
+	 *
+	 * @return array
+	 */
+	public function getNewsAndStoriesTrackingOptOut() {
+		$trackingOptOut = [];
+
+		Hooks::run( 'InstantGlobalsGetTrackingOptOut', [&$trackingOptOut] );
```

Comment:
Why do we need instant globals hook here? Can't we return simple array (better: object `{"blacklist":[...]}`?

Reply:
Good suggestion. Changed.